cmd/kpod/login.go

```diff
@@ -15,6 +16,11 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 )
 
+const (
+	// This is a default path to directory containing a CA certificate
+	CertsDir = "/etc/containers/certs.d/"
```

Comment:
Why not make use of `filepath.Join` here, This will make path string platform independent. I am sure right now we are only dealing with Linux but who knows about future or someone try to consume this library.

Reply:
I am not sure if kpod is meant to work on any other machine than a kubernetes worker! cc: @rhatdan  on this one.

Reply:
> I am not sure if kpod is meant to work on any other machine than a kubernetes worker!

If that is the case then it's fine the way you have it right now. In case if that is not scenario then you also move it to `var` instead have it `const`.

Reply:
`filepath.Join("/", "etc", …)` won’t be any more platform independent; the assumption that `/etc` exists is much more restrictive than the assumption that `/` is a working platform separator.

Reply:
We intend to support libpod/kpod on machines without kubernetes.  We want no hard requirement between kpod and cri-o other then both use conmon.